portfolio-current/server.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const EXPRESS = require('express');
+const APP = EXPRESS();
+const PORT = process.env.PORT || 3000;
+
+APP.use(EXPRESS.static('public'));
+
+APP.get('/', function(request, response){
+  response.sendFile('index.html', {root: './public'});
+});
+
+let someData = {one: 1, two: 2};
+
+APP.get('/data', function(request, response){
+  response.send(someData)
+});
```

Comment:
Just so you know, you do not need this route. It was simply built out as an example of what you can do with a server.